### PR TITLE
Add review of 2017 Starterkit

### PR DESCRIPTION
**Pull request by chrisburr**
Adds a summary of the 2017 Starterkit written by @vbellee.

### CODE REVIEW

**Comment by chrisburr** [comment]
I've added small versions (1024px wide) of all the old images and added hyperlinks to the full size versions.

**Comment by apuignav** [comment]
This looks good to go to me! I await for a second opinion before merging.

File: _posts/2017-10-30-starterkit-autumn-2017.md
Comment (alexpearce) [inline _posts/2017-10-30-starterkit-autumn-2017.md]: The date here should be that of the workshop.

**Review by alexpearce — CHANGES_REQUESTED** [review]
LGTM, just needs the date change. Thanks a _lot_!

File: _posts/2017-10-30-starterkit-autumn-2017.md
Comment (chrisburr) [reply _posts/2017-10-30-starterkit-autumn-2017.md]: Done 👍

**Review by alexpearce — APPROVED** [review]

**Pull request merged**
